.dockerignore

```diff
@@ -0,0 +1,10 @@
+**/*.local
+**/*.tsbuildinfo
+**/build
+**/node_modules
+**/target
+Dockerfile
+Dockerfile.*
+heroku.yml
+libs/fixtures/data
+libs/hmpb/fixtures
```

Comment:
Can this ignore other apps?
```
apps/admin
apps/scan
apps/central-scan
apps/mark-scan
```

Reply:
Ah, great call - worth a shot! Our pnpm workspace configuration uses glob patterns, so hopefully, there aren't any direct references to these